multiqc/utils/search_patterns.yaml

```diff
@@ -61,6 +61,10 @@ bbmap/mhist:
 bbmap/qahist:
     contents: '#Deviation'
     num_lines: 1
+bbmap/qchist:
+    fn: '*.qchist'
+    contents: '#Quality	count1	fraction1'
+    num_lines: 1
```

Comment:
I guess this is meant to be either the filename pattern _or_ the contents match? Currently it has to have both.. To make it an `OR` just turn it into a list instead:

```yaml
    - fn: '*.qchist'
    - contents: '#Quality	count1	fraction1'
      num_lines: 1
```

See https://multiqc.info/docs/#step-1-find-log-files

Reply:
Hi @ewels, yes I included the pattern search for both filename and content because the possibilty of using a custom file extension (for fname) and because BBMap qahist has the same output structure but with multiple columns; qchist instead has only those 3 columns.

Reply:
I'm not sure using a filename pattern is meaningful at all, since there is no consensus on the filenames of these output files. I typically name mine `*.qchist.txt`, but I've seen tons of different variants and pretty much never(!) met any other BBMap user that used the same naming convention as I do :D.. .